<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RetrieveVaultItem</class>
 <widget class="QWidget" name="RetrieveVaultItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RetrieveVaultItem</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Retrieve Item</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>320</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <property name="text">
    <string>Retrieve</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>321</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>OCR A Extended</family>
       </font>
      </property>
      <property name="text">
       <string>Account No</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="accountEdit">
      <property name="font">
       <font>
        <family>OCR A Extended</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>OCR A Extended</family>
       </font>
      </property>
      <property name="text">
       <string>Vault No</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="vaultIdEdit">
      <property name="font">
       <font>
        <family>OCR A Extended</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>OCR A Extended</family>
       </font>
      </property>
      <property name="text">
       <string>Item No</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="itemIdEdit">
      <property name="font">
       <font>
        <family>OCR A Extended</family>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>-130</x>
     <y>-70</y>
     <width>871</width>
     <height>701</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/guidesign/BackgroundImage.png);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>320</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="messagelabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>410</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>label_5</zorder>
  <zorder>pushButton</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>messagelabel</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="finalbankingsystem.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>RetrieveVaultItem</receiver>
   <slot>backButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>-3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>RetrieveVaultItem</receiver>
   <slot>retrieveButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>-5</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>backButtonClicked()</slot>
  <slot>retrieveButtonClicked()</slot>
 </slots>
</ui>
